README.md

```diff
@@ -24,6 +24,13 @@ At this time, only new issues & pull requests that support the Ionic Framework w
 
 This project uses [pnpm](https://pnpm.io/) for package management. You can install it by following the [installation instructions](https://pnpm.io/installation).
 
+## Requirements
+
+This project requires:
+
+- Node.js v20.10.0 or higher.
+- [pnpm](https://pnpm.io/) v9.4.0 or higher
```

Comment:
We gonna actually remember to update this if Renovate changes the engine values?

Reply:
I was thinking of adding it to the `ignoreDeps` list but then this is not a dependency and rather an version specification for an engine so I believe we won't see it popping up.

Reply:
Renovate will change the engine config. Here's an example from Stencil Core https://github.com/ionic-team/stencil/pull/5791

Reply:
I tried ignoring the engine fields and Renovate's validator said that pnpm is not a supported package manager, let's see what happens.